cmd/osctl/cmd/config.go

```diff
@@ -282,6 +283,7 @@ func init() {
 	configAddCmd.Flags().StringVar(&crt, "crt", "", "the path to the certificate")
 	configAddCmd.Flags().StringVar(&key, "key", "", "the path to the key")
 	configGenerateCmd.Flags().StringSliceVar(&additionalSANs, "additionalSANs", []string{}, "additional Subject-Alt-Names for the APIServer certificate")
+	configGenerateCmd.Flags().StringVar(&canonicalControlplaneEndpoint, "controlplaneEndpoint", "", "the canonical controlplane endpoint (IP or DNS name) and optional port (defaults to 6443)")
```

Comment:
nit: in other commands that have multi-word args we use something like `controlplane-endpoint`.